core/app/models/spree/reimbursement.rb

```diff
@@ -18,6 +18,8 @@ class IncompleteReimbursementError < StandardError; end
     accepts_nested_attributes_for :return_items, allow_destroy: true
 
     before_create :generate_number
+    before_create :calculate_total
+    after_create :recalculate_order
```

Comment:
I am not sure if a AR hook is the best place to do that calculation (that may include API requests and other funky stuff shops do to determine the order totals). Maybe we should move this into the `perform!` method.